card-data/01-character-upbringing.yaml

```
@@ -0,0 +1,264 @@
+      - Look past pain or hurt rather than soothe it
+- front: &Country
+    <<: *tfront
+    name: Country
+    desc: >
+      //"Take a deep breath of that country air!" -- Tana
+
+      You grew up among farmers, ranchers, or other folk who lived
+      off the land. You might have lived in a tight-knit community,
+      or been miles away from the nearest neighbor.
+
+      You're familiar with the plants or animals of your region.
+      You're used to putting in an honest day's work.
+
+      You've learned to approach situations via effort.
+    prompts:
+      - Endure wearying labor, physical or emotional
+      - Devise practical solutions to hard problems
+      - Understand someone's unspoken hardships
+  back:
+    <<: *tback
+    name: Country
+    desc: >
+      //"Smells like cow breakfast" -- Basler
+
+      Not everyone appreciates the necessity of hard work.
+      But sometimes, just plowing through isn't the right answer either.
+      You're used to getting down in the mud,
+      but it's important to reach for the stars as well.
+    prompts:
+      - Condemn someone for shunning hard work
+      - Balk at complex explanations or solutions
+      - Sacrifice tomorrow's hopes for today's needs
+- front: &Frontier
+    <<: *tfront
+    name: Frontier
+    desc: >
+      //"The forest demands self-sufficiency" -- Woody
+
+      You grew up at the edges of civilization, whether a harsh
+      desert, a lonely mountain, a deep forest, or somewhere
+      equally remote. You learned to fend for yourself from
+      a young age.
+
+      You can hunt, trap, or otherwise obtain your basic necessities.
+      Your life is a careful balance of trust and caution.
+
+      You've learned to approach situations through resourcefulness.
+    prompts:
+      - Come well-prepared for harsh conditions
+      - Spot what you need in the environment
+      - Anticipate danger
+  back:
+    <<: *tback
+    name: Frontier
+    desc: >
+      //"Well the forest sounds pretty demanding" -- Basler
+
+      When much of your life is preoccupied with taking care
+      of yourself and your needs, it's easy to lose sight of
+      others' wants.
+      Carefulness and selfishness are matters of degree.
+    prompts:
+      - Take something others need later for what you need now
+      - Cross a line in pursuit of your goals
+      - Imagine a threat in every shadow
+- front: &Insular
+    <<: *tfront
+    name: Insular
+    desc: >
+      //"This is all very different from the cloister" -- Emory
+
+      You grew up secluded from the rest of the world.
+      Whether you lived on a lonely mountain village,
+      a religious retreat, or in an enclave within a
+      city, the ways of the outside world are a mystery.
+
+      You've learned to approach situations through curiosity.
+    prompts:
+      - Solve a problem using your peoples' ways
+      - Ask illuminating questions
+      - Reveal hidden knowledge or exclusive experience
```

```suggestion
      - Solve a problem using your peoples' ways
      - Ask naive but insightful questions
      - Reveal hidden knowledge or exclusive experience
```
My hope with this change is to make a more approachable prompt. Illuminating is a kind of an off-putting instruction. "Quick! Be illuminating!" :-)